tests/inheritance/test_concrete_inheritance.py

```diff
@@ -56,11 +56,11 @@ def test_each_class_has_distinct_version_class(self):
         assert self.ArticleVersion.__table__.name == "article_version"
         assert self.BlogPostVersion.__table__.name == "blog_post_version"
 
-    @mark.skipif("True")
-    def test_each_object_has_distinct_version_class(self):
-        article = self.Article()
-        blogpost = self.BlogPost()
-        textitem = self.TextItem()
+    @mark.skipif("True", reason="concrete property is not supported yet")
```

Comment:
Why are we removing the reason?

Reply:
kept the tests but excluded the this test as for now we do not support concrete props